### ISSUE TITLE
TensorFlow Neural Network Problem 1 Error

### ISSUE THREAD

**Issue by alpernkanli** [udacity/sdc-issue-reports#4]

**Comment by alpernkanli** [comment]
In the problem 1, the normalized testing array is not true when (x - mean)/std, or min-max scaling, or something between them applied.

**Comment by Brok-Bucholtz** [comment]
@alpernkanli 

Thanks for letting us know!  The problem has been changed to Min-Max scaling, including the unit tests.
